maintainers/maintainer-list.nix

```diff
@@ -11004,6 +11004,11 @@
     githubId = 9870613;
     name = "Hubert Mühlhans";
   };
+  tristanpemble = {
+    email = "tpemble@gmail.com";
+    github = "tristanpemble"; 
+    name = "Tristan Pemble";
+  };
```

Comment:
Please to the `maintainers/maintainer-list.nix` edit in a separate commit `maintainers: add tristanpemble`.

Please also add `githubId` information, you can get this by checking https://api.github.com/users/tristanpemble (it should be `2466322`), [otherwise CI will fail](https://gist.github.com/GrahamcOfBorg/a229b8de82f05ac6e1106cd6c8c3d73f#file-ofborg-eval-lib-tests-L64).